build_tools/ci/cpu_comparison/input_generator.py

```diff
@@ -268,29 +274,42 @@ def generate_inputs(filename, write_dir, seed):
                     sub_tokens = tokens[2].split("x")
                     element_type = sub_tokens[-1]
 
-                    num_elements = 1
+                    nb_elements = 1
                     for i in range(len(sub_tokens) - 1):
-                        num_elements *= int(sub_tokens[i])
-                    bin_filename = os.path.join(
+                        nb_elements *= int(sub_tokens[i])
+                    bin_fn = os.path.join(
                         write_dir, name + "_input" + str(input_number) + ".bin"
                     )
-                    if re.search(r"\s", str(bin_filename)):
+                    if re.search(r"\s", str(bin_fn)):
                         raise RuntimeError(
-                            f"input {tokens[2]}={bin_filename} has a space in the filename, which is not supported"
+                            f"input {tokens[2]}={bin_fn} has a space in the filename, which is not supported"
                         )
 
-                    input_args.append(f"--input={tokens[2]}=@{bin_filename}")
+                    input_args.append(f"--input={tokens[2]}=@{bin_fn}")
                     # Each input has a distinct seed, based on its input number.
                     # This is to ensure that operands are not populated with the
                     # same values.
                     input_seed = seed + input_number
-                    write_input(
-                        bin_filename,
-                        num_elements,
-                        element_type,
-                        input_number,
-                        input_seed,
-                    )
+
+                    # Check if input_number is a key in the dictionary. If it is
```

Comment:
```suggestion
                    # Check if input_number is a key in the dictionary. If it is,
```
Not an expert on English grammar, but I think a comma is usually inserted after clauses?